packages/nouns-webapp/src/utils/vrgdaAuction.ts

```diff
@@ -0,0 +1,113 @@
+import { Provider } from '@ethersproject/abstract-provider';
+import { BigNumber, BigNumberish, Contract, ethers, utils } from 'ethers';
+import { useEffect } from 'react';
+import { INoun } from '../components/StandaloneNoun';
+import config from '../config';
+import LilVRGDAABI from '../libs/abi/LilVRGDA.json';
+
+const jsonProvider = new ethers.providers.JsonRpcProvider(
+  `https://${config.addresses.vrgda.network}.infura.io/v3/${process.env.REACT_APP_INFURA_PROJECT_ID}`,
+);
```

Comment:
Provider we use to get auction config and set of auctions (mentioned in previous comment)